### DISCUSSION TITLE
Is it required to implement "Display List" in the final project?

### DISCUSSION THREAD

**Discussion by JeedaAlhaffar** [MuhammadSulaiman001/opengl-lab#33] [Q&A] [answered]
Do we have to use display list since it's old-dated and has been deprecated in  modern OpenGL?

**Comment by MuhammadSulaiman001** [comment, marked as answer]
Hi, this is optional, you don't have to implement it, but you can! You can use legacy functions in modern context!

**Comment by JeedaAlhaffar** [comment]
I appreciate your time.
